unit/java/io/github/azagniotov/stubby4j/yaml/YAMLMergerTest.java

```diff
@@ -0,0 +1,42 @@
+package io.github.azagniotov.stubby4j.yaml;
+
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+import static com.google.common.truth.Truth.assertThat;
+import static java.nio.file.Files.exists;
+
+/**
+ * @author Abhijit Sarkar
+ */
+public class YAMLMergerTest {
+    @Test
+    public void testMerge() throws IOException, URISyntaxException {
```

Comment:
It is OK for tests to `throws Exception` if you are not testing for specific exception